General/using-cloud-services-without-console-access.md

```diff
@@ -0,0 +1,27 @@
+{{{
+  "title": "Using Cloud Services without Console Access",
+  "date": "09-15-2015",
+  "author": "Jared Ruckle",
+  "attachments": [],
+  "related-products" : [],
+  "contentIsHTML": false,
+  "sticky": false
+}}}
+
+### Audience
+
+IT Operations
+
+### CenturyLink Cloud does not offer console access to cloud servers. So now what?
+
+Some IT pros may be accustomed to managing environments with this capability. Here are a few ideas and strategies to help system administrators build and maintain environments without it.
+ 
+* **Follow proper change management for VM modifications as well as for the applications inside them.** Use [snapshots when a roll-back of VM “state” is desired](../Servers/creating-and-managing-server-snapshots.md). NOTE: this only works for VMs with less than 1 TB of storage.
```

Comment:
This only works for 'Standard' VM Types with less than 1 TB of Storage.  just want the VM type listed.